ci-operator/config/openshift/os/openshift-os-master.yaml

```diff
@@ -9,6 +9,9 @@ resources:
     requests:
       cpu: 1000m
       memory: 1Gi
```

Comment:
While we're here, might as well bump this a bit. That's going to be too low for cosa.